wordpress-mobile/WordPress-iOS
# Issue 3496 - Select media from wp media library - Part Two

Second step PR to address #3496.

The code now allows to sync the media library from the server.

To test it:

 * Create a new post and insert some images/videos.
 * Select the Media Library option on the media picker
 * If it's the first time you access it, it should start synchronising the media library from the server
 * After a while you should start to see some of media thumbnails showing.
 * Test by selecting images and videos.

Needs Review: @bummytime
